llvm/test/Feature/Repo/repo_pruning_dependents_1.ll

```diff
@@ -1,37 +1,46 @@
-; This testcase tests the program repository pruning pass.
+; This test checks that no redundant compilation members are added into the compilation.
+; The explanation of the dependents is given in the repo_pruning_dependents.ll,
+; Please refer to the file for details.
+;
+; In the test, functions 'A' and 'B' are both dependent on function 'C'. During the pruning
+; pass, only add a single TicketNode of 'C' to the 'repo.tickets' in order to avoid multiple
+; compilation members of function 'C' in the compilation.
+;
+; The steps in this test are:
+; Step 1: Remove the database.
+; Step 2: Generate an IR code targeted on Repo.
```

Comment:
“Generate an IR code targeted on Repo” -> “Generate IR targeting the Repo”.

Reply:
Fixed, thanks.